File: dongyimaiparent/dongyimaimanagerweb/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http){
    this.findAll=function (){
        return $http.get("../brand/findAll.do");
    }
    this.findPage=function (){
        return $http.get("../brand/findPage.do?page="+page+"&rows="+rows);
    }
    this.save=function (methodName,entity){
        return $http.post("../brand/"+methodName+".do",entity);
    }
    this.findone=function (id){
        return $http.get("../brand/findone.do?id="+id)
    }
    this.dele=function (selectIds){
        return $http.get("../brand/delete.do?ids="+selectIds)
    }
    this.search=function (page,rows,searchEntity){
        return $http.post("../brand/search.do?page="+page+"&rows="+rows,searchEntity)
    }
    this.select2BrandList=function (){
        return $http.get("../brand/select2BrandList.do");
    }

})

File: dongyimaiparent/dongyimaisellergoodsinterface/src/main/java/com/dongyimai/sellergoods/service/BrandService.java
package com.dongyimai.sellergoods.service;

import com.dongyimai.entity.PageResult;
import com.dongyimai.entity.Result;
import com.dongyimai.pojo.TbBrand;
import com.dongyimai.pojo.TbBrandExample;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand> findAll();

    /*定义一个展示所有品牌的分页方法*/
    public PageResult findPage(TbBrand tbBrand,int pageNum,int pageSize);


    /*添加品牌信息*/
    public int add(TbBrand tbBrand);

    /*添加修改品牌信息的方法*/
    public TbBrand findone(long id);
    public int update(TbBrand tbBrand);

    /*删除*/
    public int delete(Long ids[]);


    /*返回map集个数据*/
    public List<Map> select2BrandList();

}
